experiments/AtmosGCM/moist-baroclinic-wave-bulksfcflux.jl

```diff
@@ -0,0 +1,385 @@
+#!/usr/bin/env julia --project
+
+using ArgParse
+using LinearAlgebra
+using StaticArrays
+using Test
+
+using ClimateMachine
+using ClimateMachine.Atmos
+using ClimateMachine.Orientations
+using ClimateMachine.ConfigTypes
+using ClimateMachine.Diagnostics
+using ClimateMachine.GenericCallbacks
+using ClimateMachine.ODESolvers
+using ClimateMachine.TurbulenceClosures
+using ClimateMachine.SystemSolvers: ManyColumnLU
+using ClimateMachine.Mesh.Filters
+using ClimateMachine.Mesh.Grids
+using ClimateMachine.TemperatureProfiles
+using ClimateMachine.Thermodynamics:
+    air_density, air_temperature, total_energy, internal_energy, PhasePartition, air_pressure
+using ClimateMachine.TurbulenceClosures
+using ClimateMachine.VariableTemplates
+
+using CLIMAParameters
+using CLIMAParameters.Planet:
+    day, grav, R_d, Omega, planet_radius, MSLP, T_triple, press_triple, LH_v0, R_v
+using CLIMAParameters.Atmos.SubgridScale:
+    C_drag
+
+struct EarthParameterSet <: AbstractEarthParameterSet end
+#press_triple(::EarthParameterSet) = 610.78      # from Thatcher and Jablonowski (2016)
+#C_drag(::EarthParameterSet) = 0.0044            # "
```

Comment:
```suggestion
import CLIMAParameters
CLIMAParameters.Planet.press_triple(::EarthParameterSet) = 610.78      # from Thatcher and Jablonowski (2016)
CLIMAParameters.Atmos.SubgridScale.C_drag(::EarthParameterSet) = 0.0044            # "
```

Reply:
Ah. So do we still need lines 25-29?

Maybe update the [example](https://clima.github.io/CLIMAParameters.jl/dev/#Behavior-1)-Compile-time-constants-1) @charleskawczynski?

Reply:
Ah, yes. Will do.

Reply:
> So do we still need lines 25-29?

Partially

Reply:
Actually, let's keep them to be safe. I think we might be able to strip line 28-29, and remove `press_triple` from line 27, but it should work with those lines included.